test/keys.test.js

```diff
@@ -22,7 +22,7 @@ describe('keys', function () {
   });
 
   afterEach(function () {
-    this.application.deinit();
+    this.application?.deinit();
```

Comment:
TS in JS file?

Reply:
It is JS (and thus valid TS) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining